actors/miner/tests/miner_actor_test_wpost.rs

```diff
@@ -0,0 +1,123 @@
+use fil_actor_miner as miner;
+
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::sector::RegisteredSealProof;
+
+use fvm_ipld_bitfield::{BitField, UnvalidatedBitField};
+
+mod util;
+
+const DEFAULT_SECTOR_EXPIRATION: u64 = 220;
```

Comment:
Retain the explanation for this magic number "an expriration ~10 days greater than effective min expiration taking into account 30 days max between pre and prove commit"

Reply:
sure!

Reply:
done.